pallets/contracts/src/lib.rs

```diff
@@ -183,18 +192,38 @@ decl_module! {
             <pallet_contracts::Module<T>>::update_schedule(origin, schedule)
         }
 
+        /// Enable or disable the extrinsic `put_code` in this module.
+        ///
+        /// ## Arguments
+        /// - `origin` which must be root.
+        /// - `is_enabled` is the new value for this flag.
+        ///
+        /// ## Errors
+        /// - `BadOrigin` if caller is not root.
+        ///
+        /// ## Permissions
```

Comment:
Do we want the `Permissions` type here if we don't have any permission

Reply:
It is still a requirement from our "Team Agreement" doc